### DISCUSSION TITLE
Set DEFAULT borderColor, but retain access to other colours

### DISCUSSION THREAD

**Discussion by kempster** [tailwindlabs/tailwindcss#9022] [Help]
Reduced `tailwind.config.js` example:

```
theme: {
    colors: {
        'black': '#000000',
        'white': '#FFFFFF',
        'border-grey': '#CDCDCD',
        'yellow': '#FFFF00'
    },
    borderColor: {
        DEFAULT: '#CDCDCD',
    },
    extend: {
    }
}
```

I am trying to set a default border colour to work when `<div class="border">` is used, but want to still be able to use e.g. `<div class="border-yellow">`.

Currently I can't do this without setting each of these colours under `borderColor` in the config. Is there a way to make these available across all components automatically?

**Comment by wongjn** [comment]
Pop your `borderColor` config in `extend` and it will extend the base config, instead of override it:

```diff
 theme: {
     colors: {
         'black': '#000000',
         'white': '#FFFFFF',
         'border-grey': '#CDCDCD',
         'yellow': '#FFFF00'
     },
-    borderColor: {
-        DEFAULT: '#CDCDCD',
-    },
     extend: {
+        borderColor: {
+            DEFAULT: '#CDCDCD',
+        },
     }
 }
```
Here's a [working playground example](https://play.tailwindcss.com/WS2iJGMtfU?file=config).
